<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start_divide</class>
 <widget class="QDialog" name="start_divide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>开始分割</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>250</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="type_combo">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>image</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>video</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>nii</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>待分割数据路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="data_path_text">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_btn_1">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>分割后结果保存路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="result_path_text">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_btn_2">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>80</number>
       </property>
       <property name="leftMargin">
        <number>80</number>
       </property>
       <property name="rightMargin">
        <number>80</number>
       </property>
       <item>
        <widget class="QPushButton" name="ok_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
